src/targets/resources/normal-tree.js

```diff
@@ -1,18 +1,17 @@
-import { Resource } from "../resource.js";
+import { Tree } from "./tree.js";
 import NormalLogs from "../../items/logs/normal-logs.js";
 
-export class NormalTree extends Resource {
+export class NormalTree extends Tree {
```

Comment:
Idk if this caused it, but can't seem to chop trees. Autoclicker still works on them though

Reply:
You need an axe equipped. Does it still not work w/o an axe?

Reply:
Ah ok in RS it didn't matter whether or not it was equipped. If we keep it that way we will prob want a log telling the player they need it equipped

Reply:
Yeah I have it in trello as a task to check for that. I'll probably do it at the same time as I add a check for the axe level